loggestd/src/usage_monitor.rs

```diff
@@ -39,6 +41,47 @@ impl From<Statvfs> for SpaceData {
     }
 }
 
+#[cfg(unix)]
+fn get_fs_data(directory: &Path) -> Result<SpaceData, io::Error> {
+    Ok(statvfs(directory)
+        .map_err(|e| io::Error::new(io::ErrorKind::Other, e))?
+        .into())
+}
+
+#[cfg(windows)]
+fn get_fs_data(directory: &Path) -> Result<SpaceData, io::Error> {
+    use widestring::U16CString;
```

Comment:
Why use this over `OsString`?

Reply:
https://retep998.github.io/blog/winapi-0.3/

This page (and another blog post I saw) suggest collecting the OsStr into a vector, pushing the null termination and then get that vector as bytes. I think that I rather have a crate deal this for us